assets/page/SettingsAccount.svelte

```diff
@@ -9,18 +9,30 @@ import {getContext} from 'svelte';
 import {l} from '../js/i18n';
 import {route} from '../store/Route';
 
-const colorSchemeOptions = [['auto', 'Auto'], ['dark', 'Dark'], ['light', 'Light']];
 const user = getContext('user');
 const updateUserOp = user.api.operation('updateUser');
 
+$: calculateColorSchemeOptions(theme);
+
+function calculateColorSchemeOptions(id) {
```

Comment:
Sorry for the nitpicking, but could you move this down to the rest of the functions and in alphabetic order?

Reply:
Done